rhcephcompose/compose.py

```diff
@@ -48,6 +48,8 @@ def __init__(self, conf):
         self.extra_files = conf['extra_files']
         # Whether sources composition should be skipped
         self.include_sources = conf.get('include_sources', True)
+        # Whether -dbg composition should be skipped
```

Comment:
Just noticed this comment. It describes the opposite of the behavior; it should probably be:

> Whether -dbg composition should happen